.pre-commit-config.yaml

```diff
@@ -1,13 +1,14 @@
 repos:
-- repo: local
-  hooks:
-  - id: frontend-v2-npm-check
-    name: frontend-v2 npm-check
-    entry: bash -c 'cd frontend-v2 && npm run check' # Run check script on frontend-v2
-    language: system  # Use system's node/npm installation
-    files: ^frontend-v2/  # Only run when frontend-v2 files change
-    pass_filenames: false # Don't pass individual filenames to the script
-    stages: [pre-commit]  # Run during pre-commit stage
+# NOTE disabled until we fix npm install on macos runners
```

Comment:
@liefra @HCookie fyi I disabled this temporarily because for some reason `npm` fails to be installed on the macos self hosted runners. I contacted Iain already about it

(also please note the package-lock.json commit I did in this repo -- the npm run check was actually failing for me without it, despite having done no changes myself)